### DISCUSSION TITLE
Create-next-app Issues

### DISCUSSION THREAD

**Discussion by DanEstok** [VRSEN/agency-swarm#118] [General]
I've moved stuff around a few different times and ways and cant seem to get over the writers block on making this work. ISO some help before I actually do break my wall from bashing my head on it. I have even tried to create the next app and direct it and it still fails me.

🐧 WebDeveloper 🛠️ Executing Function
Function(arguments='{"path":"echoapp"}', name='DirectoryNavigator', output=None)
─────────────────────────────────────────────────────────────────────────────────────────────────
DirectoryNavigator ⚙️ Function Output
Successfully changed directory to: echoapp
─────────────────────────────────────────────────────────────────────────────────────────────────
🐧 WebDeveloper 🛠️ Executing Function
Function(arguments='{}', name='CheckCurrentDir', output=None)
─────────────────────────────────────────────────────────────────────────────────────────────────
CheckCurrentDir ⚙️ Function Output
You must create an app first to use this tool.
─────────────────────────────────────────────────────────────────────────────────────────────────
🐧 WebDeveloper 🛠️ Executing Function
Function(arguments='{"command":"create-next-app","options":"my-next-app"}', name='RunCommand',   
output=None)
─────────────────────────────────────────────────────────────────────────────────────────────────
RunCommand ⚙️ Function Output
Error: [WinError 2] The system cannot find the file specified

**Comment by nicko-ai** [comment]
Hey @DanEstok,

  The create-next-app command failing isn't an agency-swarm issue. Your WebDeveloper agent is trying to run Next.js commands, but that's outside our framework's scope.

  If you're building an agent that creates web apps, make sure Node.js and npm are properly installed on the system where the agent runs. The Windows error suggests npx/npm aren't in the system PATH.

  This belongs in your agent's implementation details, not as a framework issue.
